apps/aecore/lib/aecore/structures/data_tx.ex

```diff
@@ -44,10 +48,16 @@ defmodule Aecore.Structures.DataTx do
   use ExConstructor
 
   @spec init(tx_types(), payload(), binary(), integer(), integer()) :: DataTx.t()
-  def init(type, payload, sender, fee, nonce) do
+  def init(type, payload, sender, fee, nonce)
+      when sender == nil or byte_size(sender) == @pub_key_size do
```

Comment:
you can use `Wallet.get_pub_key_size()`, just define constants with `@`

Reply:
The problem is that I can't use remote functions in Guards, that's why I use constant @pub_key_size, and initialize it with the function Wallet.get_pub_key_size() in the beginning of module.

Reply:
do you see around using guards, otherwise let's keep it like this